appinfo/info.xml

```diff
@@ -26,7 +26,7 @@ Before you update to a new version, [check the changelog](https://github.com/nex
     <screenshot small-thumbnail="https://raw.githubusercontent.com/nextcloud/news/master/screenshots/2-small.png">https://raw.githubusercontent.com/nextcloud/news/master/screenshots/2.png</screenshot>
     <screenshot small-thumbnail="https://raw.githubusercontent.com/nextcloud/news/master/screenshots/3-small.png">https://raw.githubusercontent.com/nextcloud/news/master/screenshots/3.png</screenshot>
     <dependencies>
-        <php min-version="5.6"/>
+        <php min-version="7.0"/>
```

Comment:
What is this new number based on?

Reply:
Also Nextcloud 13 was the last version to support php 5.6 after that it is 7+
And our nextcloud min-version is already 14

Reply:
In that case it's fine to require 7.0 everywhere. I'll just bump it again once the feed-io usage is stable enough.